### PR TITLE
Initialize doneCh in Tello NewDriverWithIP

### PR DESCRIPTION
**Pull request by joeyberkovitz**
Partially resolves https://github.com/hybridgroup/gobot/issues/793

### CODE REVIEW

**Comment by gen2thomas** [comment]
Hi @joeyberkovitz ,  thanks for preparation of the PR. Currently we are fighting with the download problem of gobot framework since v2+, so please be patient. No action is required, also no rebase at the moment.

After that phase I will have a look to your change.

Thank you for your patience!

**Comment by gen2thomas** [comment]
Hi @joeyberkovitz , we have fixed our issue without creating this "v2" subfolder. So please rebase your change.

Thanks, Thomas

File: platforms/dji/tello/driver.go
Comment (gen2thomas) [inline platforms/dji/tello/driver.go:1]: Now both constructors equal except the IP-address, nice. My suggestion is to replace the code by "return NewDriverWithIP("192.168.10.1", port)" in  NewDriver(port){...}.

Comment (joeyberkovitz) [reply platforms/dji/tello/driver.go:1]: sounds good, done

**Comment by gen2thomas** [comment]
Hi @joeyberkovitz , thanks for your work, now merged.

**Pull request merged**